nginx/etc/confd/conf.d/nginx_site_redirect_to_https.conf.toml

```diff
@@ -1,6 +1,6 @@
 [template]
 src   = "nginx/site_redirect_to_https.conf.tmpl"
-dest  = "/etc/nginx/sites-available/default-20-site_redirect_to_https.conf"
+dest  = "/etc/nginx/sites-available/default-05-site_redirect_to_https.conf"
```

Comment:
ok, it's a bit inconsistent then, with php-nginx it being after the first location {} block that's in site_base. I'll move the healthcheck to before it too. The root only needed for things that resolve to actual files